mlir/lib/Dialect/Vector/IR/VectorOps.cpp

```diff
@@ -2391,12 +2391,28 @@ mlir::vector::isBroadcastableTo(Type srcType, VectorType dstVectorType,
   // (all leading dimensions are simply duplicated).
   int64_t lead = dstRank - srcRank;
   for (int64_t r = 0; r < srcRank; ++r) {
```

Comment:
Consider renaming `r` to `rank`

Reply:
Note that `r` represents "dimension index" rather than rank. So, this should be something like `dimIdx` instead. Let me try that.